src/events/ready/ReadyEvent.js

```diff
@@ -1,10 +1,14 @@
 const BaseEvent = require('../../utils/structures/BaseEvent');
+const startMongoConnection = require('../../utils/mongo');
+const checkForPosts = require("../../utils/listeners/reminders");
 
 module.exports = class ReadyEvent extends BaseEvent {
   constructor() {
     super('ready');
   }
   async run (client) {
     console.log(client.user.tag + ' has logged in.');
+    await startMongoConnection();  // start the database connection
+    checkForPosts(client);
```

Comment:
would rename this to 'checkForReminders' or something like that. 

The ReadyEvent is not specific to this command and it's a bit hard to tell immediately what checkForPosts refers to